docs/fastdds/dds_layer/core/policy/eprosimaExtensions.rst

```diff
@@ -389,6 +393,35 @@ List of structure members:
 * |VariableLengthDataLimits::max_partitions-api|:
   States the maximum size, in octets, of the partitions data in the local or remote participant.
 
+.. _contentfilterlimits:
+
+ContentFilterProperty::AllocationConfiguration
+""""""""""""""""""""""""""""""""""""""""""""""
+
+This structure holds the limits for content-filter related discovery information.
+See |ContentFilterProperty::AllocationConfiguration-api|.
+
+List of structure members:
+
+.. list-table::
+   :header-rows: 1
+   :align: left
+
+   * - Member Name
+     - Type
+     - Default Value
+   * - |ContentFilterProperty::AllocationConfiguration::expression_initial_size-api|
+     - ``size_t``
+     - 0
+   * - |ContentFilterProperty::AllocationConfiguration::expression_parameters-api|
+     - :ref:`resourcelimitedcontainerconfig`
+     - ``{0, 100, 1}``
+
+* |ContentFilterProperty::AllocationConfiguration::expression_initial_size-api|:
+  Preallocated size of the filter expression.
+* |ContentFilterProperty::AllocationConfiguration::expression_parameters-api|:
+  Allocation configuration for the list of expression parameters.
+
 Example
```

Comment:
I would add a note in the example stating that the `content_filter` allocations cannot be set using XML for the moment.

Reply:
Commit 4daf86a